src/core/instance/proxy.js

```diff
@@ -10,7 +10,8 @@ if (process.env.NODE_ENV !== 'production') {
     'Infinity,undefined,NaN,isFinite,isNaN,' +
     'parseFloat,parseInt,decodeURI,decodeURIComponent,encodeURI,encodeURIComponent,' +
     'Math,Number,Date,Array,Object,Boolean,String,RegExp,Map,Set,JSON,Intl,' +
-    'require' // for Webpack/Browserify
+    'require,' + // for Webpack/Browserify
+    'BigInt' // for BigInt support issue #11126
```

Comment:
let's put it up after the Intl, the comment is not really necessary

Reply:
OK~, I fixed it in commit#4241f12